api/capi/meson.build

```diff
@@ -94,6 +81,56 @@ nnstreamer_capi_dep = declare_dependency(link_with: nnstreamer_capi_lib,
   ]
 )
 
+# Create dependency for tensor_filter_single for single-shot capi
+tensor_filter_single_lib = shared_library ('tensor_filter_single_lib',
```

Comment:
Question about tensor_filter_single.c :
Is it necessary to build shared library for _single.c? How about adding this file to capi_single_new_main?

Reply:
adding `tensor_filter_single.c` to `capi_single_new_main` would expose `g_tensor_filter_single_g_type` with `libcapi-nnstreamer-single-new.so`

Reply:
You are exposing them anyway as long as the application writer has access to C/C++ shared libraries.
As long as the capi-dev provides a header that lists public-APIs only, it's fine; SDK will block functions calls to be not exposed, anyway.

So.. please provide a single .so

Reply:
done.